Web/jqueryAssignment1/assignment-3.txt

```diff
@@ -0,0 +1,30 @@
+
+//Selects all SPAN with class name: �message�
+$('span.message')
+
+//Selects first DIV in a document that has class: �box�
+$('div.box').first()
+
+//Selects all BUTTONS in a document.
+$(':button')
+
+//Selects all IMAGES having alternate text as: �hello�
+$('img[alt="hello"]')
+
+//Selects all TEXT BOXES under a div having Id = �myDiv�
+$("#myDiv :text")
+
+//Selects all TEXT BOXES with name starting by: �txt�
+$("input[name^='txt']")
```

Comment:
ACTION: This is wrong as it will select all the "input" tag (buttons, password, file etc), but we only want input-text.